.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: deploy-book
+
+# Trigger the workflow on push to main branch
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - src/**
```

Comment:
Can we also have this flow run on PRs? That's very useful for checking contributions are valid before they hit master...

Reply:
Done.